engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableMultiJoinTest.java

```diff
@@ -763,6 +760,31 @@ public void testColumnConflicts() {
         }
     }
 
+    @Test
+    public void testRenamingAndMultiColumnStrings() {
+        final Table t1 = TableTools.newTable(col("A", "a", "b"), intCol("B", 1, 2), doubleCol("C", 3.0, 4.0),
+                doubleCol("D", 10.0, 20.0));
+        final Table t2 = TableTools.newTable(col("A", "a", "b"), intCol("B", 1, 3), doubleCol("C", 5.0, 6.0),
+                doubleCol("D", 30.0, 40.0));
+
+        MultiJoinTable mj = MultiJoinTableImpl.of(
+                MultiJoinInput.of(t1, "Key1=A,Key2=B", "C1=C,D1=D"),
+                MultiJoinInput.of(t2, "Key1=A,Key2=B", "C2=C,D2=D"));
```

Comment:
Please add cases with spaces around the column names, and "==". I believe JoinAddition.parse, JoinMatch.parse, and ColumnName.parse address these, but it would be best to verify.

Reply:
Also, we can pretty much just verify that we got the expected multi join input. No need to run an actual MJ for all test cases of MJI.

Reply:
I did test these manually because I was also curious but I will formalize into a test. Good point about the MJI, can be much simpler.